docs/data-mart/county_concrete_mw.md

```diff
@@ -8,7 +8,7 @@ This table contains county-level aggregates of in-progress power generation proj
 |Subject|Column|Description|Source|Notes|
 |----|----|----|----|----|
 |Identifiers|`resource_clean`|Fuel or project type|EIA 860m or ACP||
-||`iso_region`|The ISO/RTO the project will belong to. Usually 1:1 with county, but border counties may have multiple ISO/RTOs|EIA 860m or ACP||
+||`iso_region`|The ISO/RTO or Balancing Authority the project will belong to. Usually 1 per county, but border counties may have multiple.|EIA 860m or ACP||
```

Comment:
balance authority borders?

Reply:
Ya, if there are two sets of wires in the same county, both BAs will show up. Here's [GridStatus's map](https://www.gridstatus.io/ba-map) (I think based on EIA BA:county data)